### PR TITLE
Dm 2119

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2120

## Description - what does this code do?
Code adds tagline input back in to the PE Introductionsection.  A character counter is used to let the user know that 72 characters is the limit... however they can continue typing but the character counter font color changes to red but continues counting.  We do not display the tagline in the practice Show view - only on the Practice card - where the character limit is cut off at 56 chars.

## Testing done - how did you test it/steps on how can another person can test it 
1.  Login and navigate to a Practice PE introduction page.
2.  Tagline is a required field.  Character counter displays (0/72 characters)
image

3.  Type in over 72 characters - character counter text should turn red on the 73rd character typed.
image

4.  Save the tagline.
5.  Navigate to the Home page and find the practice card for the practice you used above.
6.  Verify that only up to 56 characters are displayed on the card.
image

## Acceptance criteria
- Add tagline back into PE
- Make it required
- Implement character limit of 72 (after that they can still enter characters they’re just cut off on the card)
- Character counter text turns Red after they enter 72 characters on PE
- 56 Cut off on Card side
- Height of the card is fixed
- Make it clear that this only displays on the practice card

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/assets/javascripts/_practice_editor_utilities.es6 [hunk 1]
```diff
@@ -44,14 +44,11 @@ function initSortable(ulId) {
 }
 
 function truncateText() {
+    debugger
```
Comment (ahay-agile6) [inline app/assets/javascripts/_practice_editor_utilities.es6:47]: remove

**Review by ahay-agile6 — CHANGES_REQUESTED** [review]

File: app/views/practices/_marketplace_card.html.erb [hunk 2]
```diff
@@ -29,7 +29,7 @@
           </h4>
         <%# end %>
         <div class="grid-row margin-bottom-1">
-          <p class="multiline-ellipses-1 line-height-sans-505 font-sans-sm practice-card-tagline"><%= practice.tagline %></p>
+          <p class="multiline-ellipses-1 line-height-sans-505 font-sans-sm practice-card-tagline"><%= practice.tagline.truncate(56) unless practice.tagline == nil%></p>
```
Comment (ahay-agile6) [inline app/views/practices/_marketplace_card.html.erb:32]: can you do 
`practice.tagline&.truncate(56)`
instead? (get rid of the `unless`)

**Review by ahay-agile6 — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/_practice_editor_utilities.es6 (hunk 1 above)
Comment (bradjohnson92008) [reply app/assets/javascripts/_practice_editor_utilities.es6:47]: Got it.

File: app/views/practices/_marketplace_card.html.erb (hunk 2 above)
Comment (bradjohnson92008) [reply app/views/practices/_marketplace_card.html.erb]: Yeah - thats better way to handle nil.  Thx

File: spec/features/practice_editor/introduction/image_editor_spec.rb [hunk 3]
```diff
@@ -68,10 +68,12 @@
       end
 
       it 'should display the image and save it' do
+        debugger
```
Comment (ahay-agile6) [inline spec/features/practice_editor/introduction/image_editor_spec.rb:71]: remove

File: spec/features/practice_editor/introduction/image_editor_spec.rb [hunk 4]
```diff
@@ -68,10 +68,12 @@
...
         within('section.dm-image-editor') do
           expect(page).to have_content('Thumbnail')
           expect(page).to have_content("Choose an image to represent this practice. Use a high-quality .jpg, .jpeg, or .png file that is at least 768px wide and 432px high and less than 32MB. If you want to upload an image that features a Veteran you must have Form 3203. Waivers must be filled out with the 'External to VA' check box selected.")
           expect(page).to have_link(href: 'https://vaww.rtp.portal.va.gov/DEAN/IE/DOE/10-3203.pdf')
+          debugger
```
Comment (ahay-agile6) [inline spec/features/practice_editor/introduction/image_editor_spec.rb:76]: remove

**Review by ahay-agile6 — CHANGES_REQUESTED** [review]

**Review by ahay-agile6 — APPROVED** [review]

**Review by JoshingYou1 — CHANGES_REQUESTED** [review]
Looks great Brad! I only noticed a small thing. We aren't displaying the character count on page load. 
Screen Shot 2020-10-15 at 3 44 40 PM

**Review by emyl3 — CHANGES_REQUESTED** [review]
Great job, Brad. One thing I noticed is that the character count doesn't work on page load of the practice editor.
Screen Shot 2020-10-15 at 2 42 54 PM

**Comment by bradjohnson92008** [comment]
Updated Character Counter to display on page load.. and change text color if max chars exceeded.
image

**Review by emyl3 — APPROVED** [review]
Looks good, Brad! Thank you :)

**Review by JoshingYou1 — APPROVED** [review]

**Pull request merged**